src/topology/locally_constant/algebra.lean

```diff
@@ -110,4 +117,29 @@ instance [ring Y] : ring (locally_constant X Y) :=
 instance [comm_ring Y] : comm_ring (locally_constant X Y) :=
 { .. locally_constant.comm_semiring, .. locally_constant.ring }
 
+variables (R : Type*)
+
+instance [has_scalar R Y] : has_scalar R (locally_constant X Y) :=
+{ smul := λ r f,
+  { to_fun := r • f,
+    is_locally_constant := ((is_locally_constant f).comp ((•) r) : _), } }
+
+@[simp] lemma coe_smul [has_scalar R Y] (r : R) (f : locally_constant X Y) :
```

Comment:
If you're feeling generous, could you add `coe_add`, `coe_zero`, `coe_mul` lemmas etc above too? Right now we only have the `apply` versions.